otter/test/convergence/test_service.py

```diff
@@ -790,7 +842,7 @@ def plan(*args, **kwargs):
                                status=ScalingGroupStatus.ERROR),
              noop),
             (Log('group-status-error', dict(isError=True, cloud_feed=True,
-                                            status='ERROR')),
+                                            status='ERROR', reasons='')),
```

Comment:
Not super important but will be nice to see actual presentable reason to ensure `present_reasons` is called. I am ok without it too :)